docs/book/RELEASE

```diff
@@ -1,11 +1,25 @@
-To push the docs you must be an admin until we have proper CI around this.
-  
+*NOTE: There is an issue to fully automate the publishing of Gitbook updates:
+https://github.com/kubernetes-sigs/cluster-api/issues/655 For now these 
+instructions are still partially manually while we gain experience with
+autogenerated commits to ensure we do not publish broken documentation...*
+
+# Rebuild the Gitbook
+
 ```
-cd docs/book/
-gitbook install
-gitbook build
-cd ../../
+hack/build-gitbook.sh
+```
+
+# Verify that they appear to be correct
+
+cd docs/book
+gitbook serve
+open localhost:4000
```

Comment:
is `open` portable?

Reply:
Interesting question. On MacOS, `man open` says:

> HISTORY
     First appeared in NextStep.

On Linux `open` is actually `openvt` and is provided by the kbd project. According to it's man page the command has existed since:

> 19 Jul 1996                         V1.4                           OPENVT(1)

So while it is not POSIX, it's been around a long time. Do you think it would suffice if I added a comment explaining how to get the `open` command if it is not already installed on your system (I believe it is installed by default on MacOS...)?

Reply:
i think a comment would be easier - e.g.
`# open localhost:4000 in a browser`